frr-10.1.yaml

```diff
@@ -0,0 +1,111 @@
+package:
+  name: frr-10.1
+  version: 10.1.1
+  epoch: 0
+  description: The FRRouting Protocol Suite
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    provides:
+      - frr=${{package.full-version}}
+    runtime:
+      - python-3.11
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - elfutils-dev
+      - flex
+      - json-c-dev
+      - libcap-dev
+      - libtool
+      - libyang-dev
+      - linenoise-dev
+      - lua5.3
+      - lua5.3-dev
+      - m4
+      - openssf-compiler-options
+      - openssl-dev
+      - patch
+      - pcre2-dev
+      - protobuf-c-dev
+      - python3-dev
+      - readline-dev
+      - rtrlib-dev
+      - zeromq-dev
```

Comment:
There's nothing zeromq related in the frrrouting/frr image that I can see

Reply:
I think we can remove this because I was able to compile it without it as well. 

the reason to put it here was following, the configure script takes in few overwrites and from here I added these build time deps. 
```bash
 $ ./configure -h | grep LIBS
  LIBS        libraries to pass to the linker, e.g. -l<library>
  LUA_LIBS    linker flags for LUA, overriding pkg-config
  PYTHON_LIBS linker flags for Python
  PROTOBUF_C_LIBS
  CARES_LIBS  linker flags for CARES, overriding pkg-config
  LIBYANG_LIBS
  SQLITE3_LIBS
  SYSREPO_LIBS
  GRPC_LIBS   linker flags for GRPC, overriding pkg-config
  DPDK_LIBS   linker flags for DPDK, overriding pkg-config
  UST_LIBS    linker flags for UST, overriding pkg-config
  ZEROMQ_LIBS linker flags for ZEROMQ, overriding pkg-config
  RTRLIB_LIBS linker flags for RTRLIB, overriding pkg-config
  UNWIND_LIBS linker flags for UNWIND, overriding pkg-config
```